documentation/course/e2/README.md

```diff
@@ -298,7 +337,62 @@ TODO @everyone: Trzeba zmienić layout poniższej tabelki, tak żeby pokazywał
+
+![Diagram bazodanowy Jobberknoll](./images/database-diagram-jobberknoll.drawio.svg)
+
+<table>
+  <tr>
+    <th colspan="3">Indeksy</th>
+  </tr>
+  <tr>
+    <th>Kolumna</th>
+    <th>Typ</th>
+    <th>Opis</th>
+  </tr>
+  <tr>
+    <td><code>account.id</code></td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks tworzony automatycznie przez bazę danych.</td>
+  </tr>
+  <tr>
+    <td><code>account.account_type</code></td>
+    <td>bitmap</td>
+    <td>Indeks wspierający wyszukiwanie kont według typu.</td>
+  </tr>
+  <tr>
+    <td><code>account.email</code></td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks wspierający utrzymanie unikalności adresów e-mail oraz ich wyszukiwanie.</td>
+  </tr>
+  <tr>
+    <th colspan="3">Ograniczenia</th>
+  </tr>
+  <tr>
+    <td colspan="3"><code>account.account_type IN ('A', 'D', 'I', 'P')</code></td>
+  </tr>
+  <tr>
+    <td colspan="3"><code>account.full_name <> ''</code></td>
+  </tr>
+  <tr>
+    <td colspan="3"><code>account.email <> ''</code></td>
+  </tr>
+  <tr>
+    <td colspan="3"><code>account.phone_number IS NULL OR account.account_type = 'P'</code></td>
+  </tr>
+</table>
+
+Zdecydowano się na silnik **PostgreSQL**, ze względu na jego popularność, znajomość w zespole, wsparcie na AWS RDS oraz licencję open-source. Wykorzystano najnowszą stabilną wersję **17.2**, która zapewnia najnowsze funkcje i poprawki bezpieczeństwa. PostgreSQL nie oferuje wersji LTS, natomiast każda wersja jest wspierana przez co najmniej 5 lat[^postgres-version].
+
+Jako klasę instancji wybrano **`db.t4g.medium`**. Jest to najnowsza generacja typów i poszczególne modele różnią się praktycznie jedynie liczbą vCPU oraz pamięcią RAM. Wersja `medium` jest przeznaczona dla systemów o średnim obciążeniu i oferuje 2 vCPU oraz 4 GiB RAM. Jako typ składowania wybrano **`gp3`**, który jest najnowszym i rekomendowanym przez amazon typem generalnego przeznaczenia. Baza przechowuje istotne i wrażliwe dane, zatem kluczowe jest włączenie szyfrowania.
```

Comment:
These two points are so generic that every single one of us could use them to describe their choices. How about we put it before all of specifics and say that these are the common defaults if not specified otherwise. Same with databases on lines 340 and 342.